### PR TITLE
vmui: maximum queries

### PR DESCRIPTION
**Pull request by Loori-R**
- Set the maximum number of queries to 4
- Fixed workaround for adding more queries via URL
- Removed multi Y-axes (left one for all queries)
- "Execute Query" and "Add Query" buttons moved below query fields
- Changed icon for remove query
- Reduced query field height
- Removed dotted lines to indicate queries in the graph

Frame 1

#3169
#3101

### CODE REVIEW

File: app/vmui/packages/vmui/src/utils/query-string.ts [hunk 1]
```diff
@@ -105,7 +106,8 @@ export const getQueryStringValue = (
 
 export const getQueryArray = (): string[] => {
   const queryLength = window.location.search.match(/g\d+.expr/gmi)?.length || 1;
-  return new Array(queryLength).fill(1).map((q, i) => {
-    return getQueryStringValue(`g${i}.expr`, "") as string;
-  });
+  // .slice(0, MAX_QUERY_FIELDS)
```
Comment (dmitryk-dk) [inline app/vmui/packages/vmui/src/utils/query-string.ts:109]: Remove this comment part of the code, please

**Review by dmitryk-dk — APPROVED** [review]
LGTM! See small comment

File: app/vmui/packages/vmui/src/utils/query-string.ts (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/utils/query-string.ts]: Fixed

**Review by hagen1778 — APPROVED** [review]

**Comment by valyala** [comment]
@Loori-R , thanks for the pull request! I found a small issue with the updated functionality - when one of the query is deleted, then the graph and the legend isn't updated. Could you address this issue in a follow-up pull reuqest?

**Comment by valyala** [comment]
FYI, the issue [from this comment](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/3196#issuecomment-1269765205) has been fixed in the commit 27ed4b853edd98eca0e5859cc4b3da3ddeec407c , which will be included in the next release.

**Comment by valyala** [comment]
FYI, [VictoriaMetrics v1.82.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) properly removes lines from the graph as soon as the corresponding query is removed from vmui.

**Pull request merged**